code/modules/cm_marines/dropship_ammo.dm

```diff
@@ -410,8 +412,9 @@
 	name = "\improper AGR-59-I 'Mini-Mike'"
 	desc = "The AGR-59-I 'Mini-Mike' incendiary minirocket is a cheap and efficient means of putting hate down range AND setting them on fire! Though rockets lack a guidance package, it makes up for it in ammunition count. Can be loaded into the LAU-229 Rocket Pod."
 	icon_state = "minirocket_inc"
-	point_cost = 500
-	fire_mission_delay = 3 //high cooldown
+	point_cost = 350
+	explosion_strength = 130
+	explosion_falloff = 30
```

Comment:
I don't think it needs to change this much.

Maybe 400 pts, same falloff, and 150 strength?

But as I said in your other PR in general fire missions probably overall need to be more powerful. Only if you feel this is exceptionally better than the others is any change necessary.